egs/wsj/s5/utils/lang/make_lexicon_fst_silprob.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+ # Copyright   2018  Johns Hopkins University (author: Daniel Povey)
+#             2018  Jiedan Zhu
```

Comment:
thanks, it look great!!   The only thing I notice is, sometimes the indentation (viewed in this diff) looks a little variable, which would normally be an error in python.

Reply:
You are right, for some reason the indentation doesn't reflect correctly in the request. Let me reupdate the request. Thanks